test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -76,6 +214,325 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
+				for _, pid := range pidList {
+					cmd := []string{"/bin/bash", "-c", fmt.Sprintf("grep Cpus_allowed_list /proc/%s/status | awk '{print $2}'", pid)}
+					cpusOfovServices, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+					testlog.Infof("cpus used by ovs service %s is %s", pid, cpusOfovServices)
+					Expect(cpusOfovServices == ovnContainerCpus).To(BeTrue(), "affinity of ovn kube node pods(%s) do not match with ovservices(%s)", ovnContainerCpus, cpusOfovServices)
+					if err != nil {
+						testlog.Error(err.Error())
+					}
+				}
+				// Delete testpod1
+				testlog.Infof("Deleting pod %v", testpod1.Name)
+				deleteTestPod(testpod1)
+
+				// Check the cpus of ovnkubenode pods
+				ovnContainerCpus, err = getOvnContainerCpus(workerRTNode)
+				Expect(err).ToNot(HaveOccurred(), "Unable to fetch cpus of the containers inside ovn kubenode pods")
+				testlog.Infof("cpus used by ovn kube node pods after deleting pod %v is %v", testpod1.Name, ovnContainerCpus)
+				pidList, err = getOVSServicesPid(workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
+
+				// Verify ovs-vswitchd and ovsdb-server process affinity is updated
+				for _, pid := range pidList {
+					cmd := []string{"/bin/bash", "-c", fmt.Sprintf("grep Cpus_allowed_list /proc/%s/status | awk '{print $2}'", pid)}
+					cpusOfovServices, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+					testlog.Infof("cpus used by ovs service %s is %s", pid, cpusOfovServices)
+					Expect(cpusOfovServices == ovnContainerCpus).To(BeTrue(), "affinity of ovn kube node pods(%s) do not match with ovservices(%s)", ovnContainerCpus, cpusOfovServices)
+					if err != nil {
+						testlog.Error(err.Error())
+					}
+				}
+
+				// Delete testpod2
+				deleteTestPod(testpod2)
+			})
+
+			It("[test_id:64103] ovs process affinity still excludes guaranteed pods after reboot", func() {
+
+				// create a deployment to deploy gu pods
+				dp := newDeployment()
+				testNode := make(map[string]string)
+				testNode["kubernetes.io/hostname"] = workerRTNode.Name
+				dp.Spec.Template.Spec.NodeSelector = testNode
+				err := testclient.Client.Create(context.TODO(), dp)
+				Expect(err).ToNot(HaveOccurred(), "Unable to create Deployment")
+
+				defer func() {
+					// delete deployment
+					testlog.Infof("Deleting Deployment %v", dp.Name)
+					err := testclient.Client.Delete(context.TODO(), dp)
+					Expect(err).ToNot(HaveOccurred())
+				}()
+
+				// Check the cpus of ovn kube node pods and ovs services
+				ovnContainerCpus, err := getOvnContainerCpus(workerRTNode)
+				Expect(err).ToNot(HaveOccurred(), "Unable to fetch cpus of the containers inside ovn kubenode pods")
+				testlog.Infof("cpus used by ovn kube node pods %v", ovnContainerCpus)
+				pidList, err := getOVSServicesPid(workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
+				for _, pid := range pidList {
+					cmd := []string{"/bin/bash", "-c", fmt.Sprintf("grep Cpus_allowed_list /proc/%s/status | awk '{print $2}'", pid)}
```

Comment:
same here and in other places regarding depending on linux commands

Reply:
please note these commands are required to run on worker node.